canvas_modules/common-canvas/__tests__/common-canvas/common-canvas-test.js

```diff
@@ -185,7 +221,7 @@ function createCommonCanvas(config, canvasController, toolbarConfig, notificatio
 	const selectionChangeHandler = sinon.spy();
 	const tipHandler = sinon.spy();
 	const contextMenuConfig = null;
-	const showRightFlyout = false;
+	const param2 = canvasParams || {};
```

Comment:
Can you give this a better name than 'param2'?  canvasParameters or something would be better,